src/axom/inlet/docs/sphinx/nested_structs_nested_documentation.rst

```diff
@@ -0,0 +1,235 @@
+.. |uncheck|    unicode:: U+2610 .. UNCHECKED BOX
+.. |check|      unicode:: U+2611 .. CHECKED BOX
+
+===================================
+Nested Structs Output (nested mode)
+===================================
+
+.. list-table::
+   :widths: 25 50
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Name
+     - Description
+   * - `shapes`_
+     - 
+
+
+
+------
+shapes
+------
+
+
+
+
+--------------------
+Collection contents:
```

Comment:
I'm not sure this is helpful from a user stand-point.

Reply:
The idea is to express that the following section contains the schema for individual elements of the collection.  Would it be more helpful to make this more verbose (e.g., "Each element of the collection can contain the following:") or to remove it entirely?